stats/kafka/collector.go

```diff
@@ -155,19 +155,15 @@ func (c *Collector) pushMetrics() {
 
 	// Send the samples
 	c.logger.Debug("Kafka: Delivering...")
-
+	
+	var msgList []*sarama.ProducerMessage
 	for _, sample := range formattedSamples {
 		msg := &sarama.ProducerMessage{Topic: c.Config.Topic.String, Value: sarama.StringEncoder(sample)}
-		partition, offset, err := c.Producer.SendMessage(msg)
-		if err != nil {
-			c.logger.WithError(err).Error("Kafka: failed to send message.")
```

Comment:
Is there any way to have logging while using `c.Producer.SendMessages(msgList)` on Line 165?